docs/source/getting_started.rst

```diff
@@ -5,7 +5,10 @@ Getting started
 Once the package is installed, this will by default enforce that a user will only have *one* logged in session. This will *not* proactively logout existing sessions: only if the user
 logs in with another browser or device, the old session(s) will be closed.
 
-One can disable the single session behavior by specifying the `SINGLE_USER_SESSION` setting in `settings.py` and thus setting this value to `False` (or any other value with truthiness `False`).
```

Comment:
small reminder - in RST, the backtick is used for italics, not fixed text as in Markdown. Not sure which one you wanted to use. In RST fixed text is two backticks

Reply:
Ah yes, that was intended as a code block (Markdown). Thanks!